article/app/views/fragments/mainMedia.scala.html

```diff
@@ -42,10 +42,24 @@
 
 
                     @mainVideo.videos.caption.map { caption =>
-                        <figcaption data-link-name="Video caption link" class="caption caption--main caption--video" itemprop="description">
-                            @fragments.inlineSvg("information", "icon", List("rounded-icon", "centered-icon"))
-                            @Html(caption)
-                        </figcaption>
+                        @if(mvt.ABNewNavVariantSeven.isParticipating) {
```

Comment:
To clarify, is this solely an issue for people in this A/B test? If so would have been helpful to put this in the description.